testsuite/MDAnalysisTests/coordinates/test_tng.py

```diff
@@ -0,0 +1,186 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+import numpy as np
+import pytest
+
+from numpy.testing import (
+    assert_equal,
+    assert_allclose
+)
+import MDAnalysis as mda
+from MDAnalysis.lib.mdamath import triclinic_box
+from MDAnalysis.coordinates.TNG import HAS_PYTNG
+if HAS_PYTNG:
+    import pytng
+
+from MDAnalysisTests.datafiles import (TNG_traj, TNG_traj_gro)
+
+
+@pytest.mark.skipif(not HAS_PYTNG, reason="pytng not installed")
+class TestTNGTraj(object):
+
+    _n_atoms = 1000
+    _n_frames = 101
+    _stride = 5000
+
+    # these values all taken from GMX dump
+    _pos_frame_0_first_3_atoms = np.array([[2.53300e+00,  1.24400e+00,  3.50600e+00],
+                                           [8.30000e-01,  2.54400e+00,  3.44800e+00],
+                                           [1.09100e+00,  1.10000e-01,  3.12900e+00]])
+
+    _pos_frame_100_first_3_atoms = np.array([[4.40000e-01, 3.89000e-01, 1.37400e+00],
+                                             [1.43200e+00, 1.64900e+00, 2.93900e+00],
+                                             [2.01500e+00, 2.10300e+00, 2.65700e+00]])
+
+    _box_frame_0 = np.array([3.60140e+00, 0.00000e+00, 0.00000e+00, 0.00000e+00,
+                            3.60140e+00, 0.00000e+00,  0.00000e+00, 0.00000e+00, 3.60140e+00]).reshape(3, 3)
+
+    _box_frame_100 = np.array([3.60140e+00, 0.00000e+00, 0.00000e+00, 0.00000e+00,
+                               3.60140e+00, 0.00000e+00,  0.00000e+00, 0.00000e+00, 3.60140e+00]).reshape(3, 3)
+
+    _box_frame_100 = np.array([3.58965e+00,  0.00000e+00,  0.00000e+00,  0.00000e+00,
+                              3.58965e+00,  0.00000e+00,  0.00000e+00,  0.00000e+00,  3.58965e+00]).reshape(3, 3)
```

Comment:
iirc all our gmx tests use the same data in different formats.  Is it possible to convert these to tng format?

Reply:
I can try :)

Reply:
Done.